src/sider/dialogmoter/motebehov/MotebehovKvittering.tsx

```diff
@@ -1,63 +1,80 @@
 import React from "react";
 import {
-  finnArbeidstakerMotebehovSvar,
-  motebehovFromLatestActiveTilfelle,
+  isArbeidstakerMotebehov,
+  getUbehandletSvarOgMeldtBehov,
   sorterMotebehovDataEtterDato,
+  getMotebehovInActiveTilfelle,
 } from "@/utils/motebehovUtils";
 import { tilLesbarDatoMedArUtenManedNavn } from "@/utils/datoUtils";
-import { MotebehovVeilederDTO } from "@/data/motebehov/types/motebehovTypes";
-import { ledereUtenMotebehovsvar } from "@/utils/ledereUtils";
+import {
+  MotebehovSkjemaType,
+  MotebehovVeilederDTO,
+} from "@/data/motebehov/types/motebehovTypes";
 import { useOppfolgingstilfellePersonQuery } from "@/data/oppfolgingstilfelle/person/oppfolgingstilfellePersonQueryHooks";
 import { NarmesteLederRelasjonDTO } from "@/data/leder/ledereTypes";
 import { capitalizeAllWords } from "@/utils/stringUtils";
 import MotebehovKvitteringInnhold from "@/sider/dialogmoter/motebehov/MotebehovKvitteringInnhold";
 import { useNavBrukerData } from "@/data/navbruker/navbruker_hooks";
 import { useMotebehovQuery } from "@/data/motebehov/motebehovQueryHooks";
 import { useLedereQuery } from "@/data/leder/ledereQueryHooks";
+import { OppfolgingstilfelleDTO } from "@/data/oppfolgingstilfelle/person/types/OppfolgingstilfellePersonDTO";
+import { activeNarmesteLederForCurrentOppfolgingstilfelle } from "@/sider/oppfolgingsplan/oppfolgingsplaner/AktiveOppfolgingsplaner";
+import { BodyShort } from "@navikt/ds-react";
 
 export const arbeidsgiverNavnEllerTomStreng = (lederNavn: string | null) => {
   return lederNavn ? `${lederNavn}` : "";
 };
 
-export const setSvarTekst = (deltakerOnskerMote?: boolean) => {
+export const setSvarTekst = (
+  skjemaType: MotebehovSkjemaType | null,
+  deltakerOnskerMote?: boolean
+) => {
   switch (deltakerOnskerMote) {
     case true: {
-      return " har svart JA";
+      return skjemaType === MotebehovSkjemaType.SVAR_BEHOV
```

Comment:
Det finnes to tilfeller av møtebehov: `SVAR_BEHOV` som skjer ved dialogmøtekandidattidspunkt (uke 17) der vi spør sykmeldt og arbeidsgiver om de trenger dialogmøte. Og `MELD_BEHOV` der sykmeldt og arbeidsgiver kan når som helst i sykefraværet si ifra om at de trenger et dialogmøte.

Tidligere har vi ikke skilt noe på dette, som også har skapt forvirring for både veileder og utviklere som prøver å forstå de to ulike casene 😕